src/Plugin_Command.php

```diff
@@ -748,6 +758,57 @@ protected function get_item_list() {
 		return $items;
 	}
 
+	/**
+	 * Get the wordpress.org status of a plugin.
+	 *
+	 * @param string $plugin_name The plugin slug.
+	 *
+	 * @return string The status of the plugin, includes the last update date.
+	 */
+	protected function get_wporg_data( $plugin_name ) {
+		$data = [
+			'status'       => 'no_wp_org',
+			'last_updated' => '-',
+		];
+		if ( ! $this->check_wporg['status'] && ! $this->check_wporg['update_date'] ) {
+			return $data;
+		}
+
+		if ( $this->check_wporg ) {
+			$request       = wp_remote_head( 'https://api.wordpress.org/plugins/info/1.0/' . $plugin_name );
+			$response_code = wp_remote_retrieve_response_code( $request );
+			if ( 200 === $response_code ) {
+				$data['status'] = 'active';
+			}
+			// Just because the plugin is not in the api, does not mean it was never on .org.
+		}
+
+		$request       = wp_remote_get( "https://plugins.trac.wordpress.org/log/{$plugin_name}/?limit=1&mode=stop_on_copy&format=rss" );
```

Comment:
Should we only make this request if we're checking for the date?

Reply:
I made a better check in d7ec2741628b9370a9a3229a06a404812c86a654
The answer is, not always,

When the call to api.wp is not succesfull, we need to check the SVN to distinguish  between closed and never on .org.